lua/astrocommunity/pack/purescript/README.md

```diff
@@ -0,0 +1,36 @@
+# purescript Language Pack
+
+This plugin pack does the following:
+
+- Adds `purescript` Treesitter parsers
+- Adds `purescript-language-server` language server
+- Adds `purs-tidy` formatter
+
+## How to install
+
+```lua
+return {
+  { import = "astrocommunity.pack.purescript" },
+  {
+    "AstroNvim/astrolsp",
+    opts = {
+      config = {
+        purescriptls = {
+          settings = {
+            purescript = {
+              formatter = "purs-tidy", -- add this to enable `purs-tidy` formatting on every save
+            },
+          },
+        },
+      },
+    },
+  },
+}
+```
```

Comment:
This whole section can be removed. if the astrolsp stuff is required, add it to the pack.